include/maliput/geometry_base/road_geometry.h

```diff
@@ -168,6 +182,7 @@ class RoadGeometry : public api::RoadGeometry {
   std::vector<std::unique_ptr<Junction>> junctions_;
   std::vector<std::unique_ptr<BranchPoint>> branch_points_;
   api::BasicIdIndex id_index_;
+  std::unique_ptr<StrategyBase> strategy_;
```

Comment:
We are not calling strategy_->Init() anywhere. How do you plan on doing it?

Reply:
The changes made since last discussion made this method no longer needed.